### PR TITLE
[ Fabric ] Upgrade Fabric code + ref-app to Helm 3

### PR DESCRIPTION
**Pull request by abevers**
Signed-off-by: abevers <arnoud.bevers@accenture.com>

**Changelog**
- Update Hyperledger Fabric network code to be compatible with Helm 3 (v 1.4)
- Update Supplychain reference app to be compatible with Helm 3
- Added role/task names to `deploy-network.yaml` to be able to restart at certain parts of playbook with the `--start-at-task=${TASK_NAME}` flag
- Changed Docker tags for Fabric ref-app to be consistent with our other tagging strategy
 
**To be reviewed by**
@jagpreetsinghsasan @lakshyakumar @suvajit-sarkar 

**Linked issue**
#1019

### CODE REVIEW

File: examples/supplychain-app/README.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Build the following docker images from the respective folders and store in the d
 * supplychain_frontend:latest (from `examples/supplychain-app/supplychain-frontend/` folder)
 * supplychain_corda:springboot_latest (from `examples/supplychain-app/corda/cordApps_springBoot/` folder) for Corda
 * supplychain_corda:express_app_latest (from `examples/supplychain-app/corda/express_nodeJS/` folder) for Corda
-* supplychain_fabric:rest_server_latest (from `examples/supplychain-app/fabric/chaincode_rest_server/rest-server/` folder) for Fabric
-* supplychain_fabric:express_app_latest (from `examples/supplychain-app/fabric/express_nodeJs/` folder) for Fabric
+* supplychain_fabric/rest_server:latest (from `examples/supplychain-app/fabric/chaincode_rest_server/rest-server/` folder) for Fabric
```
Comment (sownak) [inline examples/supplychain-app/README.md:29]: these are not correct images from dockerhub

File: platforms/hyperledger-fabric/configuration/README.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ We are working on a verification document for Hyperledger Fabric.
     ```
     ansible-playbook add-organization.yaml -e "@/path/to/network-fabric.yaml" -e "add_new_org='true'"
     ```
-    Follow steps in [README](https://github.com/hyperledger-labs/blockchain-automation-framework/docs/source/operations/adding_new_org_fabric.md).
+    Follow steps in [README](https://github.com/hyperledger-labs/blockchain-automation-framework/blob/develop/docs/source/operations/adding_new_org_fabric.md).
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/README.md:71]: Add readthedocs link instead

**Review by sownak — CHANGES_REQUESTED** [review]

File: examples/supplychain-app/README.md (hunk 1 above)
Comment (abevers) [reply examples/supplychain-app/README.md:29]: Did not realise these were from the official Hyperledger Labs Docker Hub (you mean [this link](https://hub.docker.com/r/hyperledgerlabs/supplychain_fabric/tags) right?). I'll change it back

File: platforms/hyperledger-fabric/configuration/README.md (hunk 2 above)
Comment (abevers) [reply platforms/hyperledger-fabric/configuration/README.md]: Changed in [c44e8bf](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/c44e8bfc500eb45912b5fb9988085bb0eb99e50c). I also updated the readthedocs link to add the `-e "add_new_org=true"` flag to the command.

File: examples/supplychain-app/README.md (hunk 1 above)
Comment (abevers) [reply examples/supplychain-app/README.md:29]: Changed in [c44e8bf](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/c44e8bfc500eb45912b5fb9988085bb0eb99e50c)

File: docs/source/operations/adding_new_org_fabric.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ For reference, see `network-fabric-add-organization.yaml` file [here](https://gi
 The [add-new-organization.yaml](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/shared/configuration/add-new-organization.yaml) playbook is used to add a new organization to the existing network. This can be done using the following command
 
 ```
-ansible-playbook platforms/shared/configuration/add-new-organization.yaml --extra-vars "@path-to-network.yaml"
+ansible-playbook platforms/shared/configuration/add-new-organization.yaml --extra-vars "@path-to-network.yaml" -e "add_new_org=true"
```
Comment (sownak) [inline docs/source/operations/adding_new_org_fabric.md:64]: add_new_org is not needed when playbook is add-new-org

File: examples/supplychain-app/README.md (hunk 1 above)
Comment (sownak) [reply examples/supplychain-app/README.md:29]: cant see the changes

File: docs/source/operations/adding_new_org_fabric.md (hunk 3 above)
Comment (abevers) [reply docs/source/operations/adding_new_org_fabric.md]: My mistake, it is only needed when calling the `platforms/hyperledger-fabric/configuration/add-organization.yaml` directly. I've changed this in commit [95916d1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/95916d1d21a7b0cc4ed0c8ac7087c1db6ad2ce0f)

File: platforms/hyperledger-fabric/configuration/README.md (hunk 2 above)
Comment (abevers) [reply platforms/hyperledger-fabric/configuration/README.md]: See comment below for completion

File: examples/supplychain-app/README.md (hunk 1 above)
Comment (abevers) [reply examples/supplychain-app/README.md]: Probably mistake in `git reset $(git merge-base)` command. Re-added in commit [95916d1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1096/commits/95916d1d21a7b0cc4ed0c8ac7087c1db6ad2ce0f)

**Review by sownak — APPROVED** [review]

**Pull request merged**